uiplib/gui/mainGui.py

```diff
@@ -0,0 +1,103 @@
+"""Module that builds the Graphical User Interface."""
+
+from uiplib.scheduler import scheduler
+from uiplib.setWallpaper import change_background
+from uiplib.utils.utils import update_settings, check_sites
+from uiplib.gui.gallery import Gallery
+from uiplib.gui import generalTab, settingsTab
+
+
```

Comment:
PEP8 will make a new PR and it'll automatically correct these inconsistencies over all of UIP :P